tests/functional/s/simplifiable/simplifiable_if_expression.py

```diff
@@ -51,3 +51,8 @@ def is_flying_animal(an_object):
     else:
         is_flying = False
     return is_flying
+
+# https://www.reddit.com/r/learnpython/comments/y5vtrw/confused_on_pylint_message_simplifiablecondition/
```

Comment:
Do you have alerts for @ Pierre, `pylint` and other terms on all sites of the internet? You seem to be everywhere :smile:

Reply:
Just checked this one time for reddit, but I do have a google alert set 😄 (randomly firing, not very useful)

Reply:
Also following the pylint tag on stackoverflow